examples/universal-cnf/vppagent/pkg/ucnf/ucnf_nse.go

```diff
@@ -1,15 +1,94 @@
 package ucnf
 
 import (
+	"bytes"
+	"context"
+	"fmt"
+	"os"
+	"time"
+
 	"github.com/danielvladco/k8s-vnet/pkg/nseconfig"
 	"github.com/davecgh/go-spew/spew"
-	"github.com/tiswanso/examples/examples/universal-cnf/vppagent/pkg/config"
+	"github.com/gofrs/uuid"
 	"github.com/networkservicemesh/networkservicemesh/sdk/common"
 	"github.com/sirupsen/logrus"
+	"github.com/tiswanso/examples/api/ipam/ipprovider"
+	"github.com/tiswanso/examples/examples/universal-cnf/vppagent/pkg/config"
+	"golang.org/x/sync/errgroup"
+	"google.golang.org/grpc"
 	"gopkg.in/yaml.v2"
-	"os"
 )
 
+type ipamService struct {
```

Comment:
I think we should go ahead and organize this under the config package.  This type and methods should probably be in vppagent/pkg/config/ in a separate file called `config_ipam.go` and the the factory method with the dial and renew go routine function should be defined in there as well.

Reply:
done